SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+import "unreal/gdk/core_types.schema";
```

Comment:
Since this is the only file using `UnrealRPCCommandResponse` now, I wanted to move that type into this file so we don't have to include core_types.schema

Reply:
Done